corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -783,34 +806,16 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", [
         validateAllFields: function () {
```

Comment:
Is this still in use anywhere?

Reply:
not anymore. looks like we can remove it.

Reply:
removed here [75a6181](https://github.com/dimagi/commcare-hq/pull/34658/commits/75a618184ddaad6a31a08798574af69f01e6b064)